Model/DutyCalculator.php

```diff
@@ -225,7 +231,6 @@ public function handleCaseWhereDutyIsAlreadyRetrieved($apply, $quote, $address)
             $this->handleTaxApplicability($address, $apply);
             $this->_logger->debug('User wants to apply the D&T. Value of apply ::' . $apply);
             $quote->setBaseReachDuty($this->checkoutSession->getBaseReachDuty());
-            $quote->setReachDuty($this->checkoutSession->getReachDuty());
```

Comment:
could you clarify why this line is being removed, but the `$quote->setBaseReachDuty($this->checkoutSession->getBaseReachDuty());` wasn't? I'm confused what the benefit is of removing these set calls

Reply:
Yeah, sure. 
$quote->setReachDuty($this->checkoutSession->getReachDuty()); is removed because it was already done in the very first line of this handleCaseWhereDutyIsAlreadyRetrieved(....) definition. The line that was removed is a duplicate/ redundant line.